tpl-implementations/include/experimental/__p1673_bits/kokkos-kernels/mdspan_to_view_mapper_kk.hpp

```diff
@@ -124,14 +124,14 @@ auto mdspan_to_view(std::experimental::mdspan<
 */
 template<
   class ElementType,
-  std::experimental::extents<>::size_type ext0,
-  std::experimental::extents<>::size_type ext1,
+  Kokkos::extents<size_t>::size_type ext0,
+  Kokkos::extents<size_t>::size_type ext1,
   class NestedLayout,
   class Accessor>
-auto mdspan_to_view(std::experimental::mdspan<
+auto mdspan_to_view(Kokkos::mdspan<
 		    ElementType,
-		    std::experimental::extents<ext0, ext1>,
-		    std::experimental::linalg::layout_transpose<NestedLayout>,
+		    Kokkos::extents<size_t, ext0, ext1>,
+		    MDSPAN_IMPL_STANDARD_NAMESPACE::MDSPAN_IMPL_PROPOSED_NAMESPACE::linalg::layout_transpose<NestedLayout>,
```

Comment:
This code uses the namespace macros, but the code above hard-codes "Kokkos" as the namespace.  Should we consider being more consistent?  Does enabling the kokkos-kernels back-end imply that `MDSPAN_IMPL_STANDARD_NAMESPACE` is always "Kokkos"?

Reply:
FIrst this isn't actually a standard library implementation here. If (or maybe optimistically "when") we contribute this stuff I am not sure that we could move the KokkosKernels impl with it. But if we did the answer to the dependency thing is simple. std::linalg -> KokkosKernels -> KokkosCore -> std stuff not including linalg (i.e. mdspan and what not). These internal dependencies in the library are not uncommon. The same would be true for calling cuBLAS or oneMKL inside the standard implementation. Both of those also have a dependency on the standard library. Same is already true for stuff like using TBB inside the parallel algorithms implementation.